src/components/js-tracer/Canvas.js

```diff
@@ -9,16 +9,16 @@ import classNames from 'classnames';
 import {
   TIMELINE_MARGIN_LEFT,
   TIMELINE_MARGIN_RIGHT,
-} from '../../app-logic/constants';
+} from 'firefox-profiler/app-logic/constants';
 import {
   withChartViewport,
   type WithChartViewport,
-} from '../shared/chart/Viewport';
-import ChartCanvas from '../shared/chart/Canvas';
-import TextMeasurement from '../../utils/text-measurement';
-import { FastFillStyle } from '../../utils';
-import { updatePreviewSelection } from '../../actions/profile-view';
-import { BLUE_40 } from '../../utils/colors';
+} from 'firefox-profiler/shared/chart/Viewport';
```

Comment:
Hi @canova, I fixed the issue on the import, but I have no idea what to do about the problems with the linter. 

I saw in the CONTRIBUTING.md file that there are instructions for installing yarn on Linux or OS X. I'm using Windows - might that be the problem? 
When I run yarn lint, I get a bunch of error messages like this one:

`ERROR in ./.yarn/$$virtual/webpack-dev-server-virtual-e34b510136/0/cache/webpack-dev-server-npm-3.11.0-42b1303dbc-1d34457
456.zip/node_modules/webpack-dev-server/client/overlay.js
Module not found: Error: Can't resolve 'html-entities' in 'C:\Users\miet\Desktop\MozillaProfiler\profiler\.yarn\$$virtual
\webpack-dev-server-virtual-e34b510136\0\cache\webpack-dev-server-npm-3.11.0-42b1303dbc-1d34457456.zip\node_modules\webpa
ck-dev-server\client'
 @ ./.yarn/$$virtual/webpack-dev-server-virtual-e34b510136/0/cache/webpack-dev-server-npm-3.11.0-42b1303dbc-1d34457456.zi
p/node_modules/webpack-dev-server/client/overlay.js 6:15-39
 @ ./.yarn/$$virtual/webpack-dev-server-virtual-e34b510136/0/cache/webpack-dev-server-npm-3.11.0-42b1303dbc-1d34457456.zi
p/node_modules/webpack-dev-server/client?http://localhost
 @ multi ./.yarn/$$virtual/webpack-dev-server-virtual-e34b510136/0/cache/webpack-dev-server-npm-3.11.0-42b1303dbc-1d34457
456.zip/node_modules/webpack-dev-server/client?http://localhost ./src/index
`

Do you have any suggestions as to what I can try next?  Thank you.

Reply:
Just wanted to let you know I've not given up on this :-)
I've installed a Linux virtual machine and am going to try to figure it out from there!

Reply:
So I started over from the Linux VM, and when I run `yarn lint` no issues come up, but I'm not able to push it to my remote repository. This is the message I'm getting:
`Lost connection to the flow server (0 retries remaining): |Out of retries, exiting!
error Command failed with exit code 7.
info Visit https://yarnpkg.com/en/docs/cli/run for documentation about this command.
ERROR: "flow" exited with 7.
error Command failed with exit code 1.
info Visit https://yarnpkg.com/en/docs/cli/run for documentation about this command.
husky > pre-push hook failed (add --no-verify to bypass)
error: failed to push some refs to 'https://github.com/v3g4n/Outreachy.git'
`

I have no idea what else to try...

Reply:
Here, the issue is with flow. "yarn flow" is the comment you can run locally. Otherwise there's always the solution to push with --no-verify that will skip running the test before pushing

Reply:
Thank you, @julienw 
I submitted a new PR #2918

Reply:
Thanks, I'm closing this PR then. Also, you don't have to create a new PR when you need to change the code in one. You can directly update the PR by pushing more commits to the branch you pushed before. For example, if your forked remote is "origin" on your local, you can do:
```bash
git commit -am "explanation of the commit" # Doing some changes in the PR after creating one
git push origin outreachy # Replace the "outreachy" with the branch you are working. Gave this example since this is the current branch name.
# OR you may need to force push if you made changes in the previous commits:
git push origin outreachy -f
```
After pushing the code to your branch, the PR also will be updated.

Reply:
Thanks so much for the info, @canova !
I'll keep that in mind next time - this was my first ever PR :-)